src/hooks/useCapabilities.js

```diff
@@ -14,15 +14,11 @@ const useCapabilities = () => {
     () => ky.get(`capabilities?limit=${stripes.config.maxUnpagedResourceCount}&query=cql.allRecords=1 sortby resource`).json(),
   );
 
-  const memoizedCapabilitiesList = useMemo(() => {
-    return data?.capabilities || [];
-  }, [data]);
-
   const groupedCapabilitiesByType = useMemo(() => {
-    return getKeyBasedArrayGroup(data?.capabilities || [], 'type');
+    return getCapabilitiesGroupedByTypeAndResource(data?.capabilities || []);
```

Comment:
If you set the default in `getCapabilitiesGroupedByTypeAndResource()` then you don't need to provide null safety here.

Reply:
I tested this way, but it crushes saying that capabilities is undefined even if provide default to getCapabilitiesGroupedByTypeAndResource